common/desktop-exports

```diff
@@ -22,15 +22,34 @@ fi
 
 export SNAP_LAUNCHER_ARCH_TRIPLET=$ARCH
 
+if [ -d $SNAP/gnome-runtime ]; then
+  if [ ! -d $SNAP/gnome-runtime/usr ]; then
+    echo "You need to connect the snap to the gnome-runtime interface."
+    echo ""
+    echo "You can do this with those commands:"
+    echo "snap install gnome318-udt"
```

Comment:
There is an issue doing this in the "common" directory though, this is the common files used for all parts. If you want to avoid the repetition like in the Qt example, we can maybe have a 3 sandwish layer like "specific/common/specific"?